components/LinksHomepage/index.js

```diff
@@ -28,16 +29,8 @@ export default function LinksHomepage() {
     localStorage.setItem("recipes", JSON.stringify(updatedRecipes));
     setLink("");
     setTitle("");
+    setCategory("Breakfast");
```

Comment:
this line sets the state of category to "Breakfast" every time the site rerenders. i am note sure if this is what you want. If you need to get the value of your select menu - look into "document.getElementFromId() method. Reading the value of your select form could make your page more dynamic

Reply:
I changed the "Breakfast" category to an empty option and named the label. Don't mind the styling, I`ll deal with it in my next PR. Thanks for the tip with the "document.getElementFromId()" method, I looked into it, but for my purposes here,  I'm happy with my solution. :)